.github/workflows/backup-db-sanitise.yml

```diff
@@ -24,7 +24,7 @@ on:
           Name of the database server. Default is the live server. When backing up a point-in-time (PTR) server, use the full name of the PTR server. (Optional)
       exclude-tables:
         description: |
-          Comma-separated list of tables to exclude from the backup. (Optional)
+          Space-separated list of tables to exclude from the backup. (Optional)
```

Comment:
Doesn't the action expect a comma separated list? i.e. https://github.com/DFE-Digital/github-actions/blob/master/backup-postgres/action.yml#L114